### ISSUE TITLE
Fontello icon styles overwriting Wordpress theme styles

### ISSUE THREAD

**Issue by bschlagel** [scossar/wp-discourse-shortcodes#3]
Seeing an issue where the "fontello" CSS is conflicting with the CSS for our Wordpress theme, which uses Font Awesome as its standard icon set. I think it's something with generic "icon" wildcard selectors — our theme for example has ones like `[class^="icon-"], [class*=" icon-"]` and looks like Fontello has similar. When I removed the contents of the files fontello.css and fontello-embedded.css and re-installed the plugin, this conflict went away. Might help to give icon classes specific to this plugin a more specific namespace to avoid problems w/ any theme already using different icon set/rules.

**Comment by scossar** [comment]
Thanks! Look for an update tomorrow.

**Comment by scossar** [comment]
I've updated the plugin, but haven't yet added this fix. I'll let you know when it's been done.

**Comment by bschlagel** [comment]
Great thanks Simon! Just starting to play around with this a bit. I'll let you know if I have anything else to add, and will keep an eye out on Discourse meta for more! Let me know if you prefer feedback / suggestions there or directly on Github.

**Comment by scossar** [comment]
For now this is the best place for feedback. I'll be making some small changes to the plugin over the next couple of days and finishing the README.

**Comment by scossar** [comment]
The icon issue should be resolved now. Fontello icons are now prefixed with iconFT-. The plugin is only including 2 icons, heart and reply.

**Comment by scossar** [comment]
Just in case you're using the 'tile' attribute in the `discourse_topics` shortcode, I've made a change to the master branch. The tiles no longer have a fixed height, instead the default css sets their min-height to `300px`. I've removed the vertical_ellipsis.js file. It's too difficult to get it to work across different themes. On the live site I have this on it was slowing down the load time. Expect a few changes to the default css this week.
